### PR TITLE
Updated instructions for Centos-8

### PR DESCRIPTION
**Pull request by kjakkali**
#### Describe Bug or Feature
Package instructions for Centos-8.

#### Describe Your Change
Updated instructions for Centos-8

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -5,7 +5,17 @@ How to install PBS using the configure script.
 
 1. Install the prerequisite packages for building PBS.
 
-  For CentOS systems you should run the following command as root:
+  For CentOS-8 systems you should run the following command as root:
```
Comment (prakashcv13) [inline INSTALL:8]: the following line is not a command.

File: INSTALL [hunk 2]
```diff
@@ -5,7 +5,17 @@ How to install PBS using the configure script.
...
+
+    Configure and enable PowerTools repo for hwloc-devel and libedit-devel packages
```
Comment (prakashcv13) [inline INSTALL:10]: why not include the actual commands.

File: INSTALL [hunk 3]
```diff
@@ -5,7 +5,19 @@ How to install PBS using the configure script.
 
 1. Install the prerequisite packages for building PBS.
 
-  For CentOS systems you should run the following command as root:
+  For CentOS-8 systems you should run the following command as root:
```
Comment (Jitendra-Rajak) [inline INSTALL:8]: I think here we can remove command as below we have both instruction and commands.

File: INSTALL [hunk 4]
```diff
@@ -5,7 +5,19 @@ How to install PBS using the configure script.
...
+
+    Configure and enable PowerTools repo for hwloc-devel and libedit-devel packages
```
Comment (Jitendra-Rajak) [inline INSTALL:10]: This instruction should start a column before so that the dnf below can be in-lined with the other dnf.

File: INSTALL [hunk 5]
```diff
@@ -5,7 +5,19 @@ How to install PBS using the configure script.
...
+      dnf install -y dnf-plugins-core
+      dnf config-manager --set-enabled PowerTools
+
+    dnf install -y gcc make rpm-build libtool hwloc-devel \
```
Comment (Jitendra-Rajak) [inline INSTALL:14]: An instruction can be added above saying install required packages or tools.

Comment (kjakkali) [reply INSTALL:16]: Mentioned at line 11 .

**Pull request merged**